test/SIA2D_adjoint.jl

```diff
@@ -96,8 +96,8 @@ function test_adjoint_SIA2D(
 
     vecBackwardSIA2D = randn(size(H, 1), size(H, 2))
 
-    dH = Huginn.SIA2D(H, simulation, t, θ)
-    JET.@test_opt broken=true target_modules=(Sleipnir, Muninn, Huginn, ODINN) Huginn.SIA2D(H, simulation, t, θ)
+    Huginn.SIA2D!(dH, H, simulation, t, θ)
```

Comment:
dH is not defined ;)
```suggestion
    dH = copy(H)
    Huginn.SIA2D!(dH, H, simulation, t, θ)
```